lib/open_food_network/xero_invoices_report.rb

```diff
@@ -188,23 +188,23 @@ def total_taxable_products(order)
     end
 
     def total_untaxable_fees(order)
-      order.adjustments.enterprise_fee.without_tax.sum(:amount)
+      order.all_adjustments.enterprise_fee.without_tax.sum(:amount)
     end
 
     def total_taxable_fees(order)
-      order.adjustments.enterprise_fee.with_tax.sum(:amount)
+      order.all_adjustments.enterprise_fee.with_tax.sum(:amount)
     end
 
     def total_shipping(order)
-      order.adjustments.shipping.sum(:amount)
+      order.all_adjustments.shipping.sum(:amount)
     end
 
     def total_transaction(order)
-      order.adjustments.payment_fee.sum(:amount)
+      order.all_adjustments.payment_fee.sum(:amount)
     end
 
     def tax_on_shipping_s(order)
-      tax_on_shipping = order.adjustments.shipping.sum(:included_tax) > 0
+      tax_on_shipping = order.all_adjustments.shipping.sum(:included_tax) > 0
```

Comment:
Code Climate wants this to be

```suggestion
      tax_on_shipping = order.all_adjustments.shipping.sum(:included_tax).positive?
```
```suggestion
      tax_on_shipping = order.all_adjustments.shipping.sum(:included_tax) > 0
```